app/controllers/contracts_controller.rb

```diff
@@ -29,7 +41,7 @@ def create
   def show
     if current_customer || current_company
       @contract = Contract.find(params[:id])
-      render "show.html.erb"
+      render 'show.html.erb'
```

Comment:
nice with the single quotes, good to stick to that in ruby unless dubs are needed. but again this render happens automatically because of the name and rails defaults. All our restful controllers should either use render calls and not use them, but we should be consistent.